activesupport/lib/active_support/values/time_zone.rb

```diff
@@ -285,6 +301,12 @@ def utc_offset
       end
     end
 
+    # Returns the offset of this time zone from UTC in seconds,
+    # taking DST into account at the given time.
+    def utc_total_offset(time=Time.now, dst=true)
```

Comment:
@kaspth thanks for the suggestion. What about `#from_offset`, should that also follow the same convention?